fw/cache.c

```diff
@@ -2313,10 +2313,6 @@ tfw_cache_build_resp(TfwHttpReq *req, TfwCacheEntry *ce, long lifetime,
 		if (tfw_cache_build_resp_body(db, trec, it, p, ce->body_len,
 					      TFW_MSG_H2(req), stream_id))
 			goto free;
-		if (!TFW_MSG_H2(req)
-		    && test_bit(TFW_HTTP_B_CHUNKED, resp->flags)
-		    && tfw_http_msg_expand_data(it, skb_head, &g_crlf, NULL))
-			goto free;
```

Comment:
I created testing issue for this https://github.com/tempesta-tech/tempesta-test/issues/289 - could you please review it and add any test cases, which I could miss?

Reply:
It looks like all potential cases are pretty much covered in there.